### DISCUSSION TITLE
Show/hide section

### DISCUSSION THREAD

**Discussion by newmedicine** [TriliumNext/Trilium#4286] [General]
Is there a way of showing/hiding a section of a note? I can't find any way of doing this. 

I vaguely expected that I would be able to show/hide sections, or nodes in an indented list, as is common in outlining software.

Can anyone think of a way of achieving this? Or am I using the wrong paradigm? 

Thanks

Nick

**Comment by zadam** [comment]
There's no such feature, please see https://github.com/zadam/trilium/issues/3636 and https://github.com/zadam/trilium/issues/3097
